src/__snapshots__/geoclient.test.js.snap

```diff
@@ -171,168 +210,56 @@ New York, NY 10007",
-            "short_name": "260",
-            "types": [
-              "street_number",
-            ],
-          },
-          {
-            "long_name": "Broadway",
-            "short_name": "Broadway",
-            "types": [
-              "route",
-            ],
-          },
-          {
-            "long_name": "Manhattan",
-            "short_name": "Manhattan",
-            "types": [
-              "political",
-              "sublocality",
-              "sublocality_level_1",
-            ],
-          },
-          {
-            "long_name": "New York",
-            "short_name": "New York",
-            "types": [
-              "locality",
-              "political",
-            ],
-          },
-          {
-            "long_name": "New York County",
-            "short_name": "New York County",
-            "types": [
-              "administrative_area_level_2",
-              "political",
-            ],
-          },
-          {
-            "long_name": "New York",
-            "short_name": "NY",
-            "types": [
-              "administrative_area_level_1",
-              "political",
-            ],
-          },
-          {
-            "long_name": "United States",
-            "short_name": "US",
-            "types": [
-              "country",
-              "political",
-            ],
-          },
-          {
-            "long_name": "10007",
-            "short_name": "10007",
-            "types": [
-              "postal_code",
-            ],
-          },
-        ],
-        "formatted_address": "260 Broadway, New York, NY 10007, USA",
-        "geometry": {
-          "location": {
-            "lat": 40.7127042,
-            "lng": -74.00594199999999,
-          },
-          "location_type": "ROOFTOP",
-          "viewport": {
-            "northeast": {
-              "lat": 40.7140531802915,
-              "lng": -74.00459301970848,
-            },
-            "southwest": {
-              "lat": 40.7113552197085,
-              "lng": -74.00729098029149,
-            },
-          },
-        },
-        "navigation_points": [
-          {
-            "location": {
-              "latitude": 40.7127398,
-              "longitude": -74.0064308,
-            },
-            "restricted_travel_modes": [
-              "WALK",
-            ],
-          },
-          {
-            "location": {
-              "latitude": 40.713009,
-              "longitude": -74.005767,
-            },
-            "restricted_travel_modes": [
-              "DRIVE",
-            ],
-          },
-        ],
-        "place_id": "ChIJgeN7FVFbwokRBk7mnDW6Krc",
-        "plus_code": {
-          "compound_code": "PX7V+3J New York, NY, USA",
-          "global_code": "87G7PX7V+3J",
-        },
-        "types": [
-          "street_address",
-        ],
-      },
-      {
-        "address_components": [
-          {
-            "long_name": "230",
-            "short_name": "230",
```

Comment:
It's not clear to me why so much of the `address_components` disappeared, both here and elsewhere. I think we need to figure out what's going on there before merging this.